test/docker_questdb.py

```
@@ -0,0 +1,195 @@
+            self._auth_file = self._write_auth_db()
+            run_args += ['-v',
+                         f'{self._auth_file}:{_CONTAINER_ROOT}/{_AUTH_DB_NAME}:ro']
+            env['QDB_LINE_TCP_AUTH_DB_PATH'] = _AUTH_DB_NAME
+        for key, value in env.items():
+            run_args += ['-e', f'{key}={value}']
+        run_args.append(self._image)
+
+        sys.stderr.write(
+            f'Starting QuestDB container from {self._image!r} '
+            f'(auth: {self.auth}, http: {self.http})\n')
+        self._container_id = subprocess.check_output(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)